ethereum/datastructures/src/main/java/tech/pegasys/teku/datastructures/operations/AttesterSlashing.java

```diff
@@ -41,6 +42,7 @@
   private final IndexedAttestation attestation_2;
 
   @Label("sos-ignore")
+  @JsonIgnore
```

Comment:
We definitely shouldn't have JSON serialisation annotations in our core classes.

Reply:
Artifact of intense debugging. Removed.